deployment-scanner/src/main/java/org/jboss/as/server/deployment/scanner/logging/DeploymentScannerLogger.java

```diff
@@ -407,4 +408,19 @@
      */
     @Message(id = 36, value = "Deployment model operation failed. %s")
     RuntimeException deployModelOperationFailed(String msg);
+
+    @LogMessage(level = WARN)
+    @Message(id = 37, value = "%s does not exist")
+    void directoryIsNonexistent(String path);
+
+    @LogMessage(level = WARN)
+    @Message(id = 38, value = "%s is not a directory")
+    void isNotADirectory(String path);
+
+    @LogMessage(level = WARN)
+    @Message(id = 39, value = "%s is not writable")
+    void directoryIsNotWritable(String path);
+
+    @Message(id = 40, value = "Could not find a path called '%s'")
```

Comment:
This should probably be something more like; `Could not find relative-to path entry for %s`.